src/main/kotlin/com/github/jensim/megamanipulator/files/FilesOperator.kt

```diff
@@ -34,8 +34,12 @@ class FilesOperator(
         refresh("clones")
     }
 
+    fun getFile(dir: File? = null, path: String): File {
```

Comment:
I dont see the need for this.
Take a look at the [LocalRepoOperatorTest](https://github.com/jensim/mega-manipulator/blob/main/src/test/kotlin/com/github/jensim/megamanipulator/actions/localrepo/LocalRepoOperatorTest.kt)
There I mock the project, to point it's base path at a temp dir

Reply:
Good catch